astronomer/providers/sftp/sensors/sftp.py

```diff
@@ -43,17 +43,18 @@ def execute(self, context: Context) -> None:
         Logic that the sensor uses to correctly identify which trigger to
         execute, and defer execution as expected.
         """
-        self.defer(
-            timeout=timedelta(seconds=self.timeout),
-            trigger=SFTPTrigger(
-                path=self.path,
-                file_pattern=self.file_pattern,
-                sftp_conn_id=self.sftp_conn_id,
-                poke_interval=self.poke_interval,
-                newer_than=self.newer_than,
-            ),
-            method_name="execute_complete",
-        )
+        if not self.poke(context=context):
```

Comment:
Can we add a description regarding this in the PR? I went to check the code what poke actually does. But, would be nice for a reviewer to quickly check the description to understand what exactly we're doing. Also, PRs with descriptions are better to go and check the history in the future. The commit message will easily give more details than having to go and check what's happening in the PR actually :)

Reply:
Yep, sure!